source/guides/1.1/getting-started.md

```diff
@@ -122,17 +121,17 @@ And... bask in the glory of your first Hanami project at
 
 <p><img src="/images/welcome-page.png" alt="Hanami welcome page" class="img-responsive"></p>
 
-## Hanami Architecture
+## Hanami's Architecture
 
-Hanami architecture **can host several Hanami (and Rack) applications in the same Ruby process**.
+Hanami's architecture **can host several Hanami (and Rack) applications in the same Ruby process**.
 
 These applications live under `apps/`.
 Each of them can be a component of our product, such as the user facing web interface, the admin pane, metrics, HTTP API etc..
 
-All these parts are a _delivery mechanism_ to the business logic that lives under `lib/`.
-This is the place where our models are defined, and interact with each other to compose the **features** that our product provides.
+All these parts are a _delivery mechanism_ to the business logic that live under `lib/`.
```

Comment:
This should stay as `lives`, since we're talking about the business logic, which is singular :)